com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/DetermineServiceCreateUpdateServiceActionsStep.java

```diff
@@ -43,36 +44,32 @@
 import com.sap.cloud.lm.sl.common.util.JsonUtil;
 import com.sap.cloud.lm.sl.mta.handlers.ArchiveHandler;
 import com.sap.cloud.lm.sl.mta.util.PropertiesUtil;
+import org.springframework.web.client.RestClientException;
 
-@Component("determineServiceCreateUpdateActionsStep")
-public class DetermineServiceCreateUpdateServiceActionsStep extends SyncFlowableStep {
+@Component("determineServiceCreateUpdateActionsStep") public class DetermineServiceCreateUpdateServiceActionsStep extends SyncFlowableStep {
 
     private static final String LAST_SERVICE_OPERATION = "last_operation";
     private static final String SERVICE_OPERATION_TYPE = "type";
     private static final String SERVICE_OPERATION_STATE = "state";
 
-    @Inject
-    private ServiceGetter serviceInstanceGetter;
+    @Inject private ServiceGetter serviceInstanceGetter;
```

Comment:
Is it from the IntelliJ or the new formatter?

Reply:
yes

Reply:
#totallynotunderstandablecode

Reply:
[https://sagittarius.agency/blog/lame-the-overuse-of-hashtags-in-a-non-twitter-environment](https://sagittarius.agency/blog/lame-the-overuse-of-hashtags-in-a-non-twitter-environment)